.github/pull_request_template.md

```diff
@@ -1,18 +1,16 @@
-<!-- If this PR closes a JIRA ticket, make sure the title
-starts with the JIRA issue number, for example
-JP-1234: <Fix a bug>
--->
-
-Closes #
+<!-- If this PR closes a JIRA ticket, make sure the title starts with the JIRA issue number, 
+for example JP-1234: <Fix a bug> -->
 Resolves [JP-nnnn](https://jira.stsci.edu/browse/JP-nnnn)
 
-**Description**
+<!-- If this PR closes a GitHub issue, reference it here by its number -->
+Closes #
 
+<!-- describe the changes comprising this PR here -->
 This PR addresses ...
 
-Checklist
-- [ ] Tests
-- [ ] Documentation
-- [ ] Change log
-- [ ] Milestone
-- [ ] Label(s)
+**Checklist**
+- [ ] added entry in `CHANGES.rst` under the corresponding subsection
```

Comment:
Minor wording nits to several of these, but I would say:
- "added entry in `CHANGES.rst` within the relevant release section"
- "updated or added relevant tests"
- "added relevant milestone"

Reply:
updated in 4f7e4e2